Toshi/Source/Toshi/File/TTRB.cpp

```diff
@@ -1,15 +1,9 @@
 #include "ToshiPCH.h"
 #include "TTRB.h"
 
-bool Toshi::TTRB::LoadTrb(const char* param_1)
+bool Toshi::TTRB::LoadTrb(const char* fn)
 {
-	TCString str(param_1);
-
-	TFile* f = TFile::Create(str, 1);
-
-	str.~TCString();
```

Comment:
destructor of any non-pointer objects is always called automatically when the variable is out of scope. in this case calling it manually may cause undefined behaviour

Reply:
👍